dojo/jira_link/helper.py

```diff
@@ -300,8 +313,17 @@ def jira_get_resolution_id(jira, issue, status):
     return resolution_id
 
 
-def jira_change_resolution_id(jira, issue, id):
-    jira.transition_issue(issue, id)
+def jira_change_resolution_id(jira, issue, jid):
+    jira.transition_issue(issue, jid)
```

Comment:
I think this may need some error handling similar to the other actions that post something to jira.

Reply:
Sure. Will throw a quick if around it.